pkg/agent/controller/egress/egress_controller.go

```diff
@@ -183,6 +197,12 @@ type EgressController struct {
 
 	eventBroadcaster record.EventBroadcaster
 	record           record.EventRecorder
+	// Whether it should read SubnetInfo of the ExternalIPPool or not.
```

Comment:
"Whether to support non-default subnets"?

Reply:
done